src/net/p2p.rs

```diff
@@ -1346,7 +1422,7 @@ impl PeerNetwork {
         let now = get_epoch_time_secs();
         let mut relay_handles = HashMap::new();
         for (_, convo) in self.peers.iter_mut() {
-            if convo.is_outbound() && convo.is_authenticated() && convo.stats.last_handshake_time > 0 && convo.stats.last_send_time + (convo.heartbeat as u64) + self.connection_opts.neighbor_request_timeout < now {
```

Comment:
Why are we moving away from the setting and using a constant instead? should we get rid of it?

Reply:
It appears to be a merge artifact as well.  Will revert.